plugins/inputs/graylog/graylog.go

```diff
@@ -203,13 +203,13 @@ func (h *GrayLog) flatten(item map[string]interface{}, fields map[string]interfa
 		id = id + "_"
 	}
 	for k, i := range item {
-		switch i.(type) {
+		switch i := i.(type) {
 		case int:
-			fields[id+k] = i.(float64)
+			fields[id+k] = i
```

Comment:
wait, how did this ever work? the type assertion would have panicked. If the intent is to make sure we always write it as a float, it should be `float64(i)`

Reply:
I didn't :) 
Corrected to `float64(i)` - as you suggested.